pkg/composer/serviceparser/serviceparser.go

```diff
@@ -570,7 +564,11 @@ func newContainer(project *types.Project, parsed *Service, i int) (*Container, e
 	if err != nil {
 		return nil, err
 	}
+	netTypeContainer := false
 	for _, net := range networks {
+		if strings.HasPrefix(net.fullName, "container") {
```

Comment:
may be `strings.HasPrefix(net.fullName, "container:")` to avoid misuse of `container*`

Reply:
fixed